pontoon/base/static/js/translate.js

```diff
@@ -1538,6 +1538,13 @@ var Pontoon = (function (my) {
           e.preventDefault();
           signIn();
         });
+
+        $('ul.links > li#sign-in').click(function (e) {
+          console.log('Reached here 1 !');
+          e.preventDefault();
+          signIn();
```

Comment:
Unfortunately, you have to move signIn (body of function and this event handler) to main.js because we have to support login on pages like https://pontoon.mozilla.org/terminology/ etc.